CedarJavaFFI/src/interface.rs

```diff
@@ -173,7 +173,7 @@ mod test {
             },
             "principal": null,
             "action": "Action::\"view\"",
-            "resource": "Resource:\"thing\""
+            "resource": "Resource::\"thing\""
```

Comment:
Interesting case caught by this change. This test was returning Deny due to a parse error.